projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -2569,7 +2569,7 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
                 this.verticalScrollContainer.onChunkLoad.emit(this.verticalScrollContainer.state);
             });
 
-            if (this.rowEditable) {
+            if (this.rowEditable && this.rowEditingOverlay) {
```

Comment:
Would it be cleaner if this is checked in the `changeRowEditingOverlayStateOnScroll` since that's where the `this.rowEditingOverlay.collapsed` check is that causes the issue?

Reply:
Please, take a look at the change and let me know if it is OK this way.